docs/tutorials/intro_tutorial.rst

```diff
@@ -879,273 +895,208 @@ True indefinitely.
+steps to run each instantiation for. We can also set the number of
+processors to be run at the same time (default ``None`` uses all
+available processors) as well as the length of the interval after which
+the model and agent reporters of the model class collect data.
 
 In the following example, we hold the height and width fixed, and vary
-the number of agents. We tell the BatchRunner to run 5 instantiations of
-the model with each number of agents, and to run each for 100 steps.\*
+the number of agents. We tell the batch runner to run 5 instantiations
+of the model with each number of agents, and to run each for 100 steps.
 
-We have it collect the final Gini coefficient value.
+We want to keep track of
 
-Now, we can set up and run the BatchRunner:
+1. the Gini coefficient value and
+2. the individual agent’s wealth development.
 
-*The total number of runs is 245. That is 10 agents to 490 increasing by
-10, making 49 agents populations. Each agent population is then run 5
-times (49* 5) for 245 iterations
+Since for the latter changes at each time step might be interesting, we
+set ``data_collection_period=1``. By default, this parameter has value
+-1, which results in the data only being collected at the end of each
+episode.
 
-.. code:: ipython3
+Note: The total number of runs is 245 (= 49 different populations \* 5
+iterations per population). However, the resulting list of dictionaries
+will be of length 6186250 (= 250 average agents per population \* 49
+different populations \* 5 iterations per population \* 101 steps per
+iteration).
 
-    from mesa.batchrunner import BatchRunner
+.. code:: python
 
-.. code:: ipython3
+    from mesa.batchrunner import batch_run
 
-    fixed_params = {"width": 10,
-                   "height": 10}
-    variable_params = {"N": range(10, 500, 10)}
+.. code:: python
 
-    batch_run = BatchRunner(MoneyModel,
-                            variable_params,
-                            fixed_params,
-                            iterations=5,
-                            max_steps=100,
-                            model_reporters={"Gini": compute_gini})
-    batch_run.run_all()
+    params = {"width": 10, "height": 10, "N": range(10, 500, 10)}
+    
+    results = batch_run(
+        MoneyModel,
+        parameters=params,
+        iterations=5,
+        max_steps=100,
+        nr_processes=None,
+        data_collection_period=1,
+        display_progress=True,
+    )
 
 
 .. parsed-literal::
 
-    245it [03:36,  1.13it/s]
-
-
-BatchRunner has two ways to collect data.
+    245it [00:25,  9.75it/s]
 
-First, one can pass model collection via BatchRunner as seen above with
-output below.
 
-.. code:: ipython3
-
-    run_data = batch_run.get_model_vars_dataframe()
-    run_data.head()
-    plt.scatter(run_data.N, run_data.Gini)
+To further analyze the return of the ``batch_run`` function, we convert
+the list of dictionaries to a pandas data frame and print its keys.
```

Comment:
*Pandas DataFrame.

Reply:
> To do this, we again filter our large data frame, this time with a fixed number of agents and only for a specific iteration of that population. To print the results, we convert the filtered data frame to a string specifying the desired columns to print.

For this part there is no need to rename, right?

Reply:
Hmm yeah, I think there is no need to rename the rest of the usage. introducing DataFrame with its proper name once, is sufficient.

But I do need to say that, Pandas doc always refers their object as DataFrame (see https://pandas.pydata.org/docs/search.html?q=data+frame), even though the concept "data frame" exists in R, etc.

Reply:
I'll leave it as is for the moment